src/pages/docs/from-javascript.elm

```diff
@@ -119,6 +119,6 @@ strings =
   [ (Code "'abc' + '123'", Code "\"abc\" ++ \"123\"")
   , (Code "'abc'.length", Code "String.length \"abc\"")
   , (Code "'abc'.toUpperCase()", Code "String.toUpper \"abc\"")
-  , (Code "'abc' + 123", Code "\"abc\" ++ toString 123")
+  , (Code "'abc' + 123", Code "\"abc\" ++ String.fromInteger 123")
```

Comment:
@lpil this should be [`String.fromInt`](https://package.elm-lang.org/packages/elm/core/latest/String#fromInt):
```suggestion
  , (Code "'abc' + 123", Code "\"abc\" ++ String.fromInt 123")
```